cmd/commands/runtime.go

```diff
@@ -252,7 +265,13 @@ func RunRuntimeInstall(ctx context.Context, opts *RuntimeInstallOptions) error {
 
 	componentNames := getComponents(rt, opts)
 
-	token, err := createRuntimeOnPlatform(ctx, opts.RuntimeName, server, runtimeVersion, opts.IngressHost, componentNames)
+	token, err := createRuntimeOnPlatform(ctx, &model.RuntimeInstallationArgs{
+		RuntimeName:    opts.RuntimeName,
+		Cluster:         server,
+		RuntimeVersion: runtimeVersion,
+		IngressHost:    &opts.IngressHost,
```

Comment:
the reason `IngressHost` is a string point is that it's optional. If it's empty you should send `nil` instead